src/modules/IscsiClientLib.rb

```diff
@@ -200,37 +200,52 @@ def getFirmwareInfo
       bios_info["stdout"] || ""
     end
 
+    # Takes the output of 'iscsiadm' either in 'node', 'session' or 'fw' mode, i.e.
+    # output where info is seperated by '=', like
+    # iface.transport_name = tcp
+    # iface.hwaddress = 42:52:54:00:8b:6a
+    # iface.bootproto = DHCP
+    #
+    #  @param [String]   stdout (where info is seperated by '=') of iscsiadm command
+    #  @return [Hash]    command output converted to hash
+    #
+    def nodeInfoToMap (stdout)
+      retval = {}
+      return retval if stdout.empty?
+
+      stdout.split("\n").each do |row|
+        key, val = row.split("=")
+
+        key = key.to_s.strip
+        if !key.empty?
+          retval[key] = val.to_s.strip
```

Comment:
is problem if value is empty? e.g. `"test= ".split("=").map(&:strip)` cause empty second element (value here )